apps/aecore/src/aec_blocks.erl

```diff
@@ -368,27 +361,16 @@ deserialize_from_binary(Bin) ->
     end.
 
 deserialize_micro_block_from_binary(Bin, Header) ->
-    Vsn = aec_headers:version(Header),
-    case serialization_template(micro, aec_headers:height(Header), Vsn) of
-        {ok, Template} ->
-            [{txs, Txs0}, {pof, PoF0}] =
-                aeser_chain_objects:deserialize(micro_block, Vsn, Template, Bin),
-            Txs = [aetx_sign:deserialize_from_binary(Tx)
-                   || Tx <- Txs0],
-            PoF = aec_pof:deserialize(PoF0),
-            {ok, #mic_block{header = Header, txs = Txs, pof = PoF}};
-        Err = {error, _} ->
-            Err
-    end.
+    Version = aec_headers:version(Header),
```

Comment:
The validation height/Vsn is lost here. Perhaps add the comment here as well.

Are you sure this is validated elsewhere, and that we do not rely on these checks?

I am more worried about the deserialization than the serialization.